gen/flights/flightplan.py

```diff
@@ -114,6 +116,65 @@ def tot_waypoint(self) -> Optional[FlightWaypoint]:
         failed to generate. Nevertheless, we have to defend against it.
         """
         raise NotImplementedError
+    
+    @cached_property
+    def bingo_fuel(self) -> int:
+        """Bingo fuel value for the FlightPlan
+        """
+        if self.waypoints is None or self.farthest_wpt_from_arrival is None:
```

Comment:
`self.waypoints` should never be `None`. It might be empty, but it won't be `None`.